service/controller/v1/resource/node/create.go

```diff
@@ -146,7 +148,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			return microerror.Mask(err)
 		}
 
-		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("set node config status of node in guest cluster '%s' to final state", key.ClusterID(customObject)))
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("set node config status of node in guest cluster '%s' to final state", key.ClusterID(customObject)), "clusterID", clusterID)
```

Comment:
You should adapt these IMO, they have the cluster ID twice now.

Reply:
I considered it, but my thinking here was that having specific tag is separate from having something in message. I can imagine use case where you only display log messages but then want to specifically filter by tags or otherwise further inspect something. I feel it's ok to have it in both, but I'm not strict about this. If you think it should be only present once, then I'll remove it from the log message.